packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -6195,6 +6209,28 @@ export function waitForCommitToBeReady(
   return null;
 }
 
+export function getSuspendedCommitReason(
+  state: SuspendedState,
+  rootContainer: Container,
+): null | string {
+  if (state.waitingForViewTransition) {
+    return 'Waiting for the previous Animation';
```

Comment:
If the previous animation finishes before the css and images do, then technically after that we were suspended on css. But that's a nuance that would require more complexity to show.

Technically if we were suspended on CSS first and then it finishes and then we were only suspended on images we should also show two tracks but that's unnecessary detail.